website/public/img/ecs-task-startup.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@erichaberkorn When I view this image in the preview, the `T0 T1 T2` headers have some empty space below them

![Screen Shot 2021-11-03 at 2 21 44 PM](https://user-images.githubusercontent.com/1077740/140178516-690923cd-d930-4bbb-8c8f-e9432977ac1d.png)
p